packages/web-components/fast-element/src/templating/repeat.ts

```diff
@@ -188,10 +188,13 @@ export class RepeatBehavior<TSource = any> implements Behavior, Subscriber {
         const totalRemoved: SyntheticView[] = [];
         const bindView = this.bindView;
         let removeDelta = 0;
+        let removeIndex = 0;
+        let availableViews = 0;
 
         for (let i = 0, ii = splices.length; i < ii; ++i) {
             const splice = splices[i];
             const removed = splice.removed;
+            availableViews += removed.length;
```

Comment:
I wonder if we could move this down to line 207 and just grab the final length of `totalRemoved` instead of adding things along the way?